Patch-Source: https://salsa.debian.org/chromium-team/chromium/-/blob/96bdb2b923774cf54f3c7598d941df3f769b7897/debian/patches/system/rust-cbor.patch

ERROR at //build/rust/crubit/BUILD.gn:11:5: Unable to load "/chr-build/chromium-152.0.7977.64/third_party/rust-toolchain/lib/third_party/crubit/support/BUILD.gn".
    "$crubit_src_dir/support:support_cpp",
    ^------------------------------------

Patch taken from https://github.com/ungoogled-software/ungoogled-chromium/pull/3928/changes
(though clearly a more permanent solution is needed, if upstream is
planning to drop USE_CBOR_RUST..)

--- a/components/cbor/BUILD.gn
+++ b/components/cbor/BUILD.gn
@@ -12,7 +12,7 @@ buildflag_header("buildflags") {
   # TODO(crbug.com/535682335): Remove `USE_CBOR_RUST` buildflag entirely,
   # unconditionally enable Rust CBOR parser, and drop `is_cronet_build` check
   # once Cronet supports Crubit dependencies.
-  flags = [ "USE_CBOR_RUST=!$is_cronet_build" ]
+  flags = [ "USE_CBOR_RUST=$enable_cpp_api_from_rust" ]
 }
 
 component("cbor") {
@@ -37,7 +37,7 @@ component("cbor") {
     "//base",
   ]
 
-  if (!is_cronet_build) {
+  if (enable_cpp_api_from_rust) {
     public_deps = [
       "//build/rust/crubit",
       "//components/cbor/rust:cbor_rust_bindings",
@@ -63,7 +63,7 @@ source_set("unit_tests") {
     "//testing/gtest",
     "//third_party/fuzztest",
   ]
-  if (!is_cronet_build) {
+  if (enable_cpp_api_from_rust) {
     deps += [ "//components/cbor/rust:cbor_rust_unittests" ]
   }
 
